src/hooks/useAuthenticated.ts

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import { USER_SELF, DEFAULT_AVATAR } from '@/components/constants/url';
+
+type Userdata = {
+  userName: string;
+  firstName: string;
+  profilePicture: string;
+};
+
+type HooksReturnType = {
+  userData: Userdata;
+  isLoggedIn: boolean;
+  isLoading: boolean;
+};
+
+const useAuthenticated = (): HooksReturnType => {
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
```

Comment:
Could we ensure UseState is of type Boolean

Reply:
done